lib/elixir/lib/task/supervisor.ex

```diff
@@ -16,8 +16,56 @@ defmodule Task.Supervisor do
 
   The options given in the child specification are documented in `start_link/1`.
 
+  Once started, you can tasks directly under it, for example:
```

Comment:
[Minor] This sentence seems weird. Seems like it is missing a verb. Probably a typo?

Reply:
TY!